CMakeLists.txt

```diff
@@ -195,7 +213,7 @@ endif()
 
 ecbuild_add_option( FEATURE CURL
                     DESCRIPTION "Curl library for transfering data with URLs"
-                    REQUIRED_PACKAGES "CURL 7.20" )
```

Comment:
This is a revert of commit 391e6f414d, why?
This triggers some deprecation warning..